Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/NetworkSearchController.java

```diff
@@ -150,4 +150,48 @@ public String getSearchTransformedNetwork(@PathVariable("projectUnixName") Strin
         return "sites/networks/searchednetwork";
     }
 
+
+    @RequestMapping(value = "sites/networks/search", method = RequestMethod.GET)
+    public String getSearchTransformedNetwork(@RequestParam("conceptId") String conceptId,@RequestParam("projectName") List<String> projectNames,
```

Comment:
This should be a REST interface method. It doesn't return a jsp page. Take a look at the other REST controllers for what to return etc.